### PR TITLE
Fix issue 87

### PR DESCRIPTION
**Pull request by l0n3star**
Issue 87:  Fix practicalplants name properties to start with upper case letter

### CODE REVIEW

File: db/practicalplants.js [hunk 1]
```diff
@@ -105,6 +105,19 @@ function readCrops() {
       replaceValue(object, property, ['Yes', 'True'], 'true');
     });
 
+    for (const property of NAME_PROPERTIES) {
+      if (object[property]) {
+        if (/[a-z]/.test(object[property][0])) {
```
Comment (petteripitkanen) [inline db/practicalplants.js]: It is probably safer to use `letter === letter.toLowerCase()` for possible non-ASCII letters.

Comment (petteripitkanen) [reply db/practicalplants.js]: Even better `character !== character.toUpperCase()` for possible other characters.

Comment (petteripitkanen) [reply db/practicalplants.js]: Never mind, it is okay to limit the replace to only `/[a-z]/`. In the test it was already checked that these properties always start with `/[a-zA-Z]/`.

**Comment by l0n3star** [comment]
Sorry about modifying package-lock.json.  I have reverted it.

I updated code to log erroroneous objects to stderr.  Only 1 object found:

{"binomialName":"Phaseolus vulgaris","commonName":"common bean, string bean, field bean, flageolet bean, French bean, garden bean, haricot bean, pop bean, snap bean","poorNutrition":"false","hardinessZone":10,"soilTexture":["sandy","loamy","clay"],"soilPh":["neutral","alkaline"],"soilWaterRetention":["well drained"],"shade":"no shade","sun":"full sun","water":"moderate","drought":"intolerant","ecosystemNiche":[],"lifeCycle":["annual"],"herbaceousOrWoody":null,"deciduousOrEvergreen":null,"growthRate":null,"matureMeasurementUnit":null,"matureHeight":2,"matureWidth":null,"flowerType":"hermaphrodite","pollinators":[],"wind":"false","maritime":"false","pollution":"false","functions":["nitrogen fixer"],"growFrom":[],"cuttingType":[],"fertility":["self fertile"],"rootZone":null,"family":"Leguminosae","genus":"Phaseolus","salinity":null}

**Comment by l0n3star** [comment]
Thanks for feedback.  Would it be better to log to a file?  Open to
suggestions.

**Comment by petteripitkanen** [comment]
It could be clearer to log in the tests, but only if `ENABLE_LOGGING` is `true`. Then this constant would be committed as `false` and used only locally when needed.

**Comment by l0n3star** [comment]
Ah I see.  Good idea.

**Pull request merged**